src/core/UserIconGroup/UserIconGroup.test.tsx

```diff
@@ -0,0 +1,264 @@
+    'Charlie Mayfield',
+    'Peyton Pennington',
+    'Justice Harrington',
+    'Terry Rivers',
+    'Robin Mercer',
+    'Morgan Vera',
+    'Ashley Miles',
+    'Jean Mullins',
+    'Charlie Mayfield',
+    'Peyton Pennington',
+    'Justice Harrington',
+    'Terry Rivers',
+    'Robin Mercer',
+    'Morgan Vera',
+    'Ashley Miles',
+    'Jean Mullins',
+    'Charlie Mayfield',
+    'Peyton Pennington',
+    'Justice Harrington',
+    'Terry Rivers',
+    'Robin Mercer',
+    'Morgan Vera',
+    'Ashley Miles',
+    'Jean Mullins',
+    'Charlie Mayfield',
+    'Peyton Pennington',
+    'Justice Harrington',
+    'Terry Rivers',
+    'Robin Mercer',
+    'Morgan Vera',
+    'Ashley Miles',
+    'Jean Mullins',
+    'Charlie Mayfield',
+    'Peyton Pennington',
+    'Justice Harrington',
+    'Terry Rivers',
+    'Robin Mercer',
+    'Morgan Vera',
+    'Ashley Miles',
+    'Jean Mullins',
+    'Charlie Mayfield',
+    'Peyton Pennington',
+    'Justice Harrington',
+    'Terry Rivers',
+    'Robin Mercer',
+    'Morgan Vera',
+    'Ashley Miles',
+    'Jean Mullins',
+    'Charlie Mayfield',
+    'Peyton Pennington',
+    'Justice Harrington',
+    'Terry Rivers',
+    'Robin Mercer',
+    'Morgan Vera',
+    'Ashley Miles',
+    'Jean Mullins',
+    'Charlie Mayfield',
+    'Peyton Pennington',
+    'Justice Harrington',
+    'Terry Rivers',
+    'Robin Mercer',
+    'Morgan Vera',
+    'Ashley Miles',
+    'Jean Mullins',
+    'Charlie Mayfield',
+    'Peyton Pennington',
+    'Justice Harrington',
+  ];
+  return userNames
+    .slice(0, length)
+    .map((name) => <UserIcon key={name} title={name} />);
+}
+
+it('should render in its most basic state', () => {
+  const { container } = render(
+    <UserIconGroup iconSize='medium'>{generateUserIcons(7)}</UserIconGroup>,
+  );
+  const userGroup = container.querySelector(
+    '.iui-user-icon-list.iui-stacked',
+  ) as HTMLElement;
+  expect(userGroup).toBeTruthy();
+  expect(userGroup.classList).not.toContain(`iui-animated`);
+  const userGroupIconCount = container.querySelectorAll(
+    '.iui-user-icon-list > .iui-user-icon',
+  );
+  expect(userGroupIconCount.length).toBe(6);
+  const countIcon = container.querySelector(
+    '.iui-user-icon-count',
+  ) as HTMLElement;
+  expect(countIcon.textContent).toBe('2');
+  expect(userGroup.querySelectorAll('.iui-stroke').length).toBeGreaterThan(0);
+  expect(userGroup.querySelectorAll('.iui-initials').length).toBeGreaterThan(0);
+});
+
+it('should render animated', () => {
+  const { container } = render(
+    <UserIconGroup iconSize='medium' animated={true}>
+      {generateUserIcons(7)}
+    </UserIconGroup>,
+  );
+  expect(
+    container.querySelector('.iui-user-icon-list.iui-animated') as HTMLElement,
+  ).toBeTruthy();
+});
+
+it('should render many icons', () => {
+  const { container } = render(
+    <UserIconGroup iconSize='medium'>{generateUserIcons(105)}</UserIconGroup>,
+  );
+  const userGroup = container.querySelector(
```

Comment:
no need to set it as separate constant, as you ar enot using it anywhere else.. can straight away check if truthy

Reply:
same in other places